classes/Init.php

```
@@ -39,55 +41,115 @@ public static function getIdsOfPages(){
+     * @return array Modified links array.
+     */
     public static function add_settings_link_to_plugin_list($links)
     {
         $links[] = '<a href="' . self::get_settings_page_url() . '">Settings</a>';
         return $links;
     }
 
+    /** Returns the full URL to the plugin settings page in the WordPress admin. */
     public static function get_settings_page_url()
     {
         return esc_url(get_admin_url(null, 'options-general.php?page=' . self::get_settings_page_relative_path()));
```

PHPStan (2.2.2)

[error] Call to an undefined static method Mawiblah\Init::get_settings_page_relative_path().


(staticMethod.notFound)